internal/host/validator.go

```diff
@@ -762,8 +758,8 @@ func (v *validator) validatePacketLossForRole(host *models.Host, clusterRoleReqs
 				if _, ok := failedHostIPs[l3.RemoteIPAddress]; !ok {
 					hostname, role, err := GetHostnameAndRoleByIP(l3.RemoteIPAddress, hosts)
 					if err != nil {
-						v.log.Warnf("unable to determine host's role and hostname for IP: %s", err)
-						continue
+						v.log.Warnf("unable to determine role and hostname for IP %s: %v", l3.RemoteIPAddress, err)
+						return ValidationPending, nil, fmt.Errorf("unable to determine role and hostname for IP %s: %s", l3.RemoteIPAddress, err)
```

Comment:
Why in case of error we return Pending ?

Reply:
Error can only come if the `GetHostnameAndRoleByIP` cannot resolve the IP. In this case, we don't want to continue the execution because it means there is at least one host for which the inventory has not yet been updated to reflect the new IP. 
Leaving it as Pending makes sense when you identify all hosts in your inventory through the IP.

Reply:
Changed it to Failure based on @gamli75 's comment in #2187